cmd/main.go

```diff
@@ -43,6 +43,7 @@ func main() {
 		os.Getenv("JENKINS_URL"),
 		os.Getenv("JENKINS_USER"),
 		os.Getenv("JENKINS_PASSWORD"),
+		&jobcontrol.JenkinsJobServer{},
```

Comment:
It should be fine to construct the `Jenkins` object manually when testing instead of using the `NewJenkins` function. I'm not sure if adding this to the public signature of `jobcontrol.NewJenkins` is an improvement. It brings the question of control over JenkinsJobServer, and if the user needs any control there.